spec/requests/members_controller_spec.rb

```diff
@@ -57,6 +57,12 @@
       end
 
       it { compare_static("/mp.php?mpn=Barnaby_Joyce&mpc=New_England&house=representatives") }
+
+      it "should redirect to the senator's page even if the gid is incorrect" do
```

Comment:
Yay tests! :smile: 

I wonder if we shouldn't explain in the test description why rather than just saying, "incorrect". I can imagine seeing this in the future and wondering, "why on earth would you do that?"